### ISSUE TITLE
Options for managing Night shift or F.lux?

### ISSUE THREAD

**Issue by niksmac** [herrbischoff/awesome-osx-command-line#165]
Can we add options for managing Sierra Night shift or F.lux?

**Comment by herrbischoff** [comment]
If you have some, please feel free to open a pull request. Just take care of the correct sorting and placement.

**Comment by herrbischoff** [comment]
Just to add my recent findings here after poking around with DTrace:

In Sierra, the process responsible for Night Shift is `/System/Library/PrivateFrameworks/AmbientDisplay.framework/Versions/A/XPCServices/com.apple.AmbientDisplayAgent.xpc/Contents/MacOS/com.apple.AmbientDisplayAgent` which directly writes ICC profiles for the display(s) to `/Library/ColorSync/Profiles/Displays`. There are apparently no configuration files like `.plist` files that hold the information. The process makes calls to a private framework called `CoreBrightness`.

Long story short: https://github.com/jenghis/nshift
